Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_12.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### O365 - Security And Compliance - Content Search v2
+
+- Fixed an issue where MFA accounts would authenticate using a deprecated application id.
```

Comment:
remember to add breaking changes statement here and the BC json file, as we discussed.